src/libs/ajax.js

```diff
@@ -238,6 +238,24 @@ const User = signal => ({
 
   postNpsResponse: async body => {
     return fetchRex('npsResponses/create', _.mergeAll([authOpts(), jsonBody(body), { signal, method: 'POST' }]))
+  },
+
+  getNihStatus: async () => {
+    try {
+      const res = await fetchOrchestration('api/nih/status', _.merge(authOpts(), { signal }))
+      return res.json()
+    } catch (error) {
```

Comment:
Is there a better way to express this special handling of 404? This feels a bit wordy to me.

Reply:
My impression is we usually don't do the error handling directly in the ajax call. Perhaps you could pull the try catch into where you call it in profile in `loadNihStatus()`, and then directly do `setNihStatus({})` if you get a 404 error?

Reply:
You're absolutely right. I keep trying to do things like this here when it should be up to the caller.